### PR TITLE
docs: updated getting started section for cloud

### PR DESCRIPTION
**Pull request by alelthomas**
## Pull request description 
updated step 2 of getting started docs

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/docs/articles/step2-installing-agent.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Step 2 - Installing the Testkube Agent
```
Comment (aabedraba) [inline docs/docs/articles/step2-installing-agent.md:1]: the instruction here is for creating another environment, but free accounts can only have 1 env max: https://testkube.io/pricing

Can you please do the instruction for when someone is creating a new account completely (to simulate that, just create a new org and you'll have the welcome page).

File: docs/sidebars.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ const sidebars = {
       },
       items: [
         "articles/step1-installing-cli",
-        "articles/step2-installing-cluster-components",
+        "articles/step2-installing-agent",
```
Comment (aabedraba) [inline docs/sidebars.js:36]: let's not just change the url for now, please, this will have cascading effects.. 
happy to keep it as cluster-components

**Review by aabedraba — CHANGES_REQUESTED** [review]

**Comment by alelthomas** [comment]
What do I do with this section btw?
CleanShot 2023-06-12 at 16 04 47

File: docs/docs/index.mdx [hunk 3]
```diff
@@ -127,5 +127,5 @@ With Testkube, tests are part of a cluster's state and can be executed as needed
 
 - ✨Automatically on deployment of annotated/labeled Kubernetes objects (services, pods, etc)
 - ⏲️ On a schedule
-- 🧑‍💻 Manually via Testkube's CLI or Open Source Dashboard
+- 🧑‍💻 Manually via Testkube's CLI or via Testkube's Dashboard
```
Comment (aabedraba) [inline docs/docs/index.mdx:130]: 👍🏽

**Review by aabedraba — APPROVED** [review]

File: docs/docs/articles/step2-installing-cluster-components.md [hunk 4]
```diff
@@ -1,33 +1,53 @@
...
+![Fill in Env Name](../img/fill-in-env-name.png)
+
+3. Copy the Helm install command into your terminal to install the environment and deploy the Testkube agent in your cluster: 
+
+![Copy Helm Command](../img/copy-helm-command.png)
+
+4. Run the command in your terminal and wait for Testkube to detect the connection.
+
+You will need *Helm* installed and `kubectl` configured with access to your Kubernetes cluster: 
+- To install `helm` just follow the [install instrcutions on the Helm web site](https://helm.sh/docs/intro/install/).
```
Comment (jfermi) [inline docs/docs/articles/step2-installing-cluster-components.md:26]:
```suggestion
- To install `helm` just follow the [install instructions on the Helm web site](https://helm.sh/docs/intro/install/).
```

**Review by jfermi — COMMENTED** [review]
One spelling correction.

**Pull request merged**